lib/index.js

```diff
@@ -276,17 +276,24 @@ module.exports = class SVGSpritemapPlugin {
         // Clean up the spritemap chunk
         compiler.hooks.emit.tapAsync(plugin, (compilation, callback) => {
             const { output: outputOptions } = this.options;
+            const assets = compilation.assets;
+            
 
             if ( outputOptions.chunk.keep ) {
                 return callback();
             }
 
             compilation.chunks.forEach((chunk) => {
+                
                 if ( chunk.name !== outputOptions.chunk.name ) {
                     return;
                 }
 
-                delete compilation.assets[chunk.files[0]];
+                if ( typeof chunk.files[1] !== 'undefined' && chunk.files[1].indexOf('.map') !== -1 ) {
```

Comment:
`chuck.files[1]` can be the actual .svg output, so need to make sure it's a sourcemap file at this index before removing.

Reply:
🤔 In that case it might be better to just check for the entire filename as `[1]` might as well be a `.map` file for another file...